Assets/Editor/ModelInspector.cs

```diff
@@ -20,26 +20,36 @@ internal static class Styles
             public static GUIStyle preSliderThumb = "preSliderThumb";
         }
 
-        private PreviewRenderUtility m_PreviewUtility;
-        
-        private Material m_ShadedPreviewMaterial;
-        private static Material m_MeshMultiPreviewMaterial;
-        private Material m_WireMaterial;
-        private static Material m_LineMaterial;
+        internal class PreviewSettings
+        {
+            public DisplayMode displayMode = DisplayMode.Shaded;
+            public int activeUVChannel = 0;
+            public bool drawWire = true;
+            
+            public Vector3 orthoPosition = new Vector3(0.0f, 0.0f, 0.0f);
+            public Vector2 previewDir = new Vector2(0, 0);
+            public float zoomFactor = 1.0f;
+            public int checkerTextureMultiplier = 10;
+
+            public Material shadedPreviewMaterial = new Material(Shader.Find("Standard"));
```

Comment:
Creating the shadedPreviewMaterial, wireMaterial, lineMaterial by default is not a great idea, since most likely that will leak memory (nothing will clean them up).